### PR TITLE
added an image gallery to Home page

### PR DESCRIPTION
**Pull request by marinacod**
## Description

Added a responsive image gallery to Home page, made the images clickable, so each image is a link to specific search result 

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|  ✓ | :sparkles: New feature     |
|    | :hammer: Refactoring       |
|    | :100: Add tests            |
|    | :link: Update dependencies |
|    | :scroll: Docs              |

### After

image

## Testing Steps / QA Criteria

Choose one of the popular destination you are interested in
Click the image
It should bring you to the search result page where you can see the hotels from the chosen destination

### CODE REVIEW

File: src/components/PopularDestinationGrid.js
Comment (madebymiguel) [inline src/components/PopularDestinationGrid.js:1]: Needs some margin bottom for when its in mobile view. Footer covers some of it

Comment (madebymiguel) [reply src/components/PopularDestinationGrid.js:1]: Also needs for a more visual indicator that the user can click the grid item. Probably a hover effect with a thin color border

**Review by madebymiguel — CHANGES_REQUESTED** [review]
Overall looks good

**Review by madebymiguel — APPROVED** [review]

**Review by madebymiguel — APPROVED** [review]

**Review by OxanaMl — APPROVED** [review]
Looks good

**Review by alton-michaux — APPROVED** [review]
Looks good @marinacod 👍🏾

**Pull request merged**